docs/tutorials/create-a-pallet.md

```diff
@@ -56,25 +72,7 @@ homepage = 'https://substrate.dev'
 license = 'Unlicensed'
 name = 'test-pallet'
 repository = 'https://github.com/substrate-developer-hub/substrate-pallet-template/'
-version = '2.0.0'
-```
-
-Update the `Cargo.toml` file in the root directory of the template node to include a reference to
```

Comment:
why was this step removed? You would still need to add a new pallet to your cargo.toml file, no?

Reply:
indeed